### ISSUE TITLE
Help/FAQ additions

### ISSUE THREAD

**Issue by alampros** [kactus-io/kactus#55]
It would be helpful to have some very basic (and probably obvious) workflow questions addressed on the [help page](https://kactus.io/help/). For example, it's not entirely clear what should *actually* be tracked by git and how Kactus changes your workflow on a project. As a new user introducing Kactus to a teams' workflow, the following [very obvious] questions need answers:

1. **Do we commit `.sketch` files to the repo?** My current understanding is that they should be ignored and we commit only the "exploded" versions. Is this the recommended strategy?

2. **How do we merge changes?** When another team member has committed changes to a project, how do we effectively receive those changes and merge them into the sketch file? Do we pull and "regenerate sketch file"?

3. **How can we leverage shared symbols?** What are they? Caveats? Will the introduction of [Sketch Libraries](https://sketchapp.com/learn/documentation/libraries/) affect Kactus?

4. **What happens when things go awry?** In a typical project, git provides a safety net that allows you to `git checkout .` or `git reset --hard` if the sh*t hits the fan. Does this change for kactus-managed sketch repos?

Some of these are address in the screen gifs on the help page, but they are super fast and hard to follow (and not searchable).

**Comment by mathieudutour** [comment]
yep, I'm planning to work on that but I'm a bit stretch right now.

If you are in a mood to write some documentation, you can do so here: https://github.com/kactus-io/website/tree/master/help. I'd happily accept any contribution :)

To answer your questions:
1) yes, `.sketch` should be ignored since it's impossible to merge them and will create conflicts. When you create a new repo from kactus, it will create a `.gitignore` with the following line: `*.sketch`. For existing repo, it doesn't yet but there is an issue tracking this: https://github.com/kactus-io/kactus/issues/41

2) I'd recommend using the [git flow](https://www.atlassian.com/git/tutorials/comparing-workflows#gitflow-workflow) process: you have your `master` branch which represent the current source of truth. When someone works on the project, they:
  * create a new branch
  * commit their changes
  * push them to GitHub (or any server)
  * open a pull request
  * the other members of the project review the changes, approve them
  * merge the pull request to the master branch
Then, everybody needs to `pull` the master before starting a new branch so that they have the latest changes. When you pull with Kactus, it will automatically regenerate the sketch files

3) Shared pages are kind of similar to Sketch Libraries but with a few distinctions:
  * the dependencies from one project to a shared page is explicit
  * you will be able to share some pages on an online registry. This will unlock open source dependencies for design! issue to track progress here: https://github.com/kactus-io/kactus/issues/13
  * it is not native, using Sketch Libraries is probably simpler and nicer. I think you can use Kactus with Sketch Libraries but I haven't tested yet
The only caveat is that you can't put images in a shared page

4) same, if something is broken, delete the sketch file, `git reset --hard` and regenerate the sketch file.

**Comment by alampros** [comment]
Thanks for the quick response! I'll submit a PR in a few.

P.S. - Should documentation issues be submitted to the [kactus-io/website](https://github.com/kactus-io/website) repo?

**Comment by mathieudutour** [comment]
yeah, I don't think GitHub handle closing referenced issues across repo yet

**Comment by alampros** [comment]
Closed in [kactus-io/website](https://github.com/kactus-io/website)
